eng/pipelines/aggregate-reports.yml

```diff
@@ -86,31 +86,14 @@ jobs:
         artifactName: coverage
         targetPath: eng/jacoco-test-coverage/target/site/
 
-    - script: |
-        git clone https://github.com/JonathanGiles/DependencyChecker.git
-        mkdir input && cp eng/DependencyChecker/client_dependencies.json input/dependencies.json
-      displayName: 'Download dependency checker'
-
-    - task: Maven@3
-      displayName: 'Analyze dependencies'
-      inputs:
-        mavenPomFile: 'DependencyChecker/core/pom.xml'
-        options: '-Dexec.args="-showall -dependencymanagement"'
-        mavenOptions: '$(MemoryOptions) $(LoggingOptions)'
-        javaHomeOption: 'JDKVersion'
-        jdkVersionOption: '1.11'
-        jdkArchitectureOption: 'x64'
-        publishJUnitResults: false
-        goals: 'clean package exec:java'
-
     - pwsh: |
         $(Build.SourcesDirectory)/eng/DependencyGraph/gen-dep-graph-data.ps1
       displayName: Generate Interdependency Report Data
 
     - pwsh: |
         copy -r target/staging $(Build.ArtifactStagingDirectory)
         copy eng/code-quality-reports/src/main/resources/index.html $(Build.ArtifactStagingDirectory)
-        copy output/dependencies.html (Join-Path $(Build.ArtifactStagingDirectory) "staging")
+        copy output/pom.client.html (Join-Path $(Build.ArtifactStagingDirectory) "staging")
```

Comment:
You will need to update the link in /eng/code-quality-reports/src/main/resources/index.html to change based on this file rename.

Reply:
Should I remove the link out to the SpotBugs aggregate report as well since that doesn't get generated anymore?